osbs/build/build_request.py

```diff
@@ -1112,6 +1114,11 @@ def render(self, validate=True):
             self.template['metadata'].setdefault('labels', {})
             self.template['metadata']['labels']['koji-task-id'] = str(koji_task_id)
 
+        filesystem_koji_task_id = self.spec.filesystem_koji_task_id.value
```

Comment:
We also need to set the `from_task_id` parameter for the `add_filesystem` plugin. 
See `render_add_filesystem` in this class.